app/models/compute_resource.rb

```diff
@@ -141,6 +141,10 @@ def start_vm(uuid)
     find_vm_by_uuid(uuid).start
   end
 
+  def start_with_cloudinit(uuid,user_data=nil)
```

Comment:
This will never get called as it's being overridden in `compute_resources/foreman/model/ovirt.rb`